### DISCUSSION TITLE
CUYUNIVERSE #11 SEM4 Trouble di Prisma

### DISCUSSION THREAD

**Discussion by akurazi** [deaafrizal/forum#25] [Troubleshooting Room] [answered]
Nanya bang ini kenapa waktu menambahkan anime_title enggak bisa ya bang.
mohon pencerahannya

![1](https://github.com/deaaprizal/forum/assets/90604448/8b123028-385f-4e57-8570-d6ba1bc7295e)
![2](https://github.com/deaaprizal/forum/assets/90604448/35130219-ce2f-48c6-a4d9-1834bacd0861)

**Comment by deaafrizal** [comment, marked as answer]
coba di migrate ulang bro, atau cobain hapus String? nya terus migrate ulang. 
cek juga kiriman variabel data yang ditangkep api coba di console log dulu.

**Reply by akurazi** [reply]
uda bg uda aman, waktu pengiriman data gak berurut dia bg, thnks bg
